backend/app/adapter/db/url.go

```diff
@@ -96,6 +97,73 @@ WHERE "%s"=$1;`,
 	return url, nil
 }
 
+// GetByAliases finds URLs for a list of aliases
+func (u URLSql) GetByAliases(aliases []string) ([]entity.URL, error) {
+	// create a string for aliases parameters
+	// it looks like: $1, $2, $3, ......
+	params := make([]string, 0, len(aliases))
+	for i := range aliases {
+		params = append(params, fmt.Sprintf("$%d", i+1))
+	}
+	parameterStr := strings.Join(params, ", ")
```

Comment:
Let's extract this to a standalone function called `composeParamList(numParams int) string`

Reply:
addressed comment